clang/lib/AST/ByteCode/Floating.h

```diff
@@ -17,63 +17,80 @@
 #include "clang/AST/APValue.h"
 #include "llvm/ADT/APFloat.h"
 
+// XXX This is just a debugging help. Setting this to 1 will heap-allocate ALL
+// floating values.
+#define ALLOCATE_ALL 0
+
```

Comment:
Do you intend to retain this long-term or is this just until you're confident in the implementation?

Reply:
I didn't have any plan for it, I just used it while implementing and thought it might come in handy later.